ironfish/src/rpc/routes/chain/getBlockInfo.ts

```diff
@@ -42,19 +62,41 @@ router.register<typeof GetBlockInfoRequestSchema, GetBlockInfoResponse>(
   GetBlockInfoRequestSchema,
   async (request, node): Promise<void> => {
     const hash = Buffer.from(request.data.hash, 'hex')
-    const header = await node.chain.getHeader(hash)
 
+    const header = await node.chain.getHeader(hash)
     if (!header) {
       throw new ValidationError(`No block with hash ${request.data.hash}`)
     }
 
+    const block = await node.chain.getBlock(header)
+    if (!block) {
+      throw new ValidationError(`No block with hash ${request.data.hash}`)
+    }
+
+    const transactions: GetBlockInfoResponse['block']['transactions'] = []
+
+    await block.withTransactionReferences(async () => {
+      for (const tx of block.transactions) {
+        const fee = await tx.transactionFee()
+
+        transactions.push({
+          transactionSignature: tx.transactionSignature().toString('hex'),
+          transactionHash: tx.transactionHash().toString('hex'),
+          transactionFee: fee.toString(),
+          spends: tx.spendsLength(),
+          notes: tx.notesLength(),
+        })
+      }
+    })
+
     request.status(200).end({
       block: {
         graffiti: header.graffiti.toString('hex'),
         hash: request.data.hash,
         previousBlockHash: header.previousBlockHash.toString('hex'),
         sequence: Number(header.sequence),
         timestamp: header.timestamp.valueOf(),
+        transactions: transactions,
```

Comment:
Again, might pitch the object property value shorthand here as well.
```suggestion
        transactions,
```

Reply:
In reference to the other conversation here, https://github.com/iron-fish/ironfish/pull/336#discussion_r691532460